ingest/ingest_pipeline.py

```diff
@@ -576,21 +563,27 @@ def main() -> None:
     Returns:
         None
     """
-    parsed_args = create_parser().parse_args()
-    validate_arguments(parsed_args)
-    arguments = vars(parsed_args)
-    # Initialize global variables for current ingest job
-    config.init(
-        arguments["study_id"],
-        arguments["study_file_id"],
-        arguments["user_metrics_uuid"],
-    )
-    ingest = IngestPipeline(**arguments)
-    status, status_cell_metadata = run_ingest(ingest, arguments, parsed_args)
-    # Print metrics properties
-    metrics_dump = config.get_metric_properties().get_properties()
-    for key in metrics_dump.keys():
-        print(f'{key}: {metrics_dump[key]}')
+    status = 0
+    try:
+        parsed_args = create_parser().parse_args()
+        validate_arguments(parsed_args)
+        arguments = vars(parsed_args)
+        # Initialize global variables for current ingest job
+        config.init(
+            arguments["study_id"],
+            arguments["study_file_id"],
+            arguments["user_metrics_uuid"],
+        )
+        ingest = IngestPipeline(**arguments)
+        status, status_cell_metadata = run_ingest(ingest, arguments, parsed_args)
+        # Print metrics properties
+        metrics_dump = config.get_metric_properties().get_properties()
+        for key in metrics_dump.keys():
+            print(f'{key}: {metrics_dump[key]}')
+    except Exception as e:
+        # ToDo ingest.props["errorType"] = "subsample:"
```

Comment:
Please note ticket for TODO in this code comment.

Reply:
comment was pre-existing to this PR